maventa/MaventaConfig.ts

```diff
@@ -0,0 +1,16 @@
+import { ProcessUtils } from '../ProcessUtils';
+
+ProcessUtils.initEnvFromDefaultFiles();
+
+const isTesting = process.env.IS_TESTING === 'true';
+
+const MaventaConfig = {
+  baseUrl: isTesting ? 'https://ax-stage.maventa.com' : 'https://ax.maventa.com',
+  clientId: process.env.CLIENT_ID!,
+  clientSecret: process.env.CLIENT_SECRET!,
+  scope: process.env.SCOPE!,
+  vendorApiKey: process.env.VENDOR_API_KEY!,
+  companyEDI: process.env.COMPANY_EDI!,
+};
+
+export { MaventaConfig };
```

Comment:
This should be split as an interface of configuration parameters provided to the constructor of the client interface.

The default values should be provided from a constants file like constants/maventa.ts. This way the end result code when built will only have the values we use, not the whole object.